### ISSUE TITLE
Flag: extern

### ISSUE THREAD

**Issue by steveklabnik** [steveklabnik/rustdoc#135]
This issue is part of https://github.com/steveklabnik/rustdoc/issues/125

## `--extern`: specify a dependency's location

Using this flag looks like this:

```bash
$ rustdoc src/lib.rs --extern lazy-static=/path/to/lazy-static
```

Similar to `--library-path`, `--extern` is about specifying the location
of a dependency. `--library-path` provides directories to search in, `--extern`
instead lets you specify exactly which dependency is located where.

----------------------------------------------------

Should we keep this flag or not?

**Comment by steveklabnik** [comment]
This is very important!

**Comment by euclio** [comment]
I don't think this is needed if we depend on cargo. cc #155

**Comment by steveklabnik** [comment]
That's true; I was thinking in the "cargo calls rustdoc" mode, not "rustdoc calls cargo"
